blaze/compute/tests/test_pandas_compute.py

```diff
@@ -680,6 +680,21 @@ def test_str_lower():
     assert_series_equal(expected, result)
 
 
+class TestStrCat():
+    ds = dshape('3 * {name: string[10], comment: string[25], num: int32}')
```

Comment:
Can you make one of these string columns variable length (`string` rather than `string[25]`)?